### PR TITLE
Add CLI to blueprint overview section

### PR DESCRIPTION
**Pull request by minhtuev**
Add CLI commands to the blueprint overview section

### CODE REVIEW

**Comment by minhtuev** [comment]
image

File: source/blueprints/overview.rst [hunk 1]
```diff
@@ -68,7 +68,14 @@ Depending on which blueprint you choose to run, this Quick Start should take bet
 1. Download
```
Comment (ktick) [inline source/blueprints/overview.rst:68]: After the line "Depending on which blueprint you choose to run, this Quick Start should take between five and fifteen minutes to complete.", add "You can use either use the MindMeld command line interface (insert hyperlink to https://www.mindmeld.com/docs/userguide/getting_started.html#command-line-interfaces) or a Python shell to set up the blueprint".

File: source/blueprints/overview.rst [hunk 2]
```diff
@@ -68,7 +68,14 @@ Depending on which blueprint you choose to run, this Quick Start should take bet
...
 ^^^^^^^^^^^
 
-Open a Python shell and type the following commands to download and set up the blueprint application of your choice.
```
Comment (ktick) [inline source/blueprints/overview.rst:71]: Type the following commands to download and set up the blueprint application of your choice. Use the appropriate blueprint name as the value of the ``bp_name`` variable (either ``banking_assistant``, ``hr_assistant``, ``food_ordering``, ``home_assistant``, ``video_assistant``, or ``kwik_e_mart``). In these examples, we specify the Food Ordering blueprint.

File: source/blueprints/overview.rst [hunk 3]
```diff
@@ -68,7 +68,14 @@ Depending on which blueprint you choose to run, this Quick Start should take bet
...
+Using command-line interface (CLI): use command `mindmeld blueprint [blueprint name]` to retrieve the blueprint. For example:
```
Comment (ktick) [inline source/blueprints/overview.rst:71]:
```suggestion
Using CLI: use the command `mindmeld blueprint [bp_name]` to retrieve the blueprint.
```

File: source/blueprints/overview.rst [hunk 4]
```diff
@@ -68,7 +68,14 @@ Depending on which blueprint you choose to run, this Quick Start should take bet
...
+
+.. code:: console
+
+   mindmeld blueprint food_ordering
+
+
+Use Python: use the following snippet to download and set up the blueprint application of your choice.
 
   - Use the appropriate Python blueprint name as the value of the ``bp_name`` variable (either ``banking_assistant``, ``hr_assistant``, ``food_ordering``, ``home_assistant``, ``video_assistant``, or ``kwik_e_mart``). In the example, we specify the Food Ordering blueprint.
```
Comment (ktick) [inline source/blueprints/overview.rst:80]:
```suggestion
```

File: source/blueprints/overview.rst [hunk 5]
```diff
@@ -68,7 +68,14 @@ Depending on which blueprint you choose to run, this Quick Start should take bet
...
 
-Open a Python shell and type the following commands to download and set up the blueprint application of your choice.
+Using command-line interface (CLI): use command `mindmeld blueprint [blueprint name]` to retrieve the blueprint. For example:
+
+.. code:: console
+
+   mindmeld blueprint food_ordering
+
+
+Use Python: use the following snippet to download and set up the blueprint application of your choice.
```
Comment (ktick) [inline source/blueprints/overview.rst:78]:
```suggestion
Using Python: use the following snippet to retrieve the blueprint.
```

File: source/blueprints/overview.rst [hunk 6]
```diff
@@ -84,6 +91,11 @@ Open a Python shell and type the following commands to download and set up the b
 
 Build the Natural Language Processing models that power the app.
 
+.. code:: console
```
Comment (ktick) [inline source/blueprints/overview.rst:94]:
```suggestion
Using CLI:

.. code:: console
```

File: source/blueprints/overview.rst [hunk 7]
```diff
@@ -84,6 +91,11 @@ Open a Python shell and type the following commands to download and set up the b
...
+
+   python -m food_ordering build
+
+
 .. code:: python
```
Comment (ktick) [inline source/blueprints/overview.rst:99]:
```suggestion
Using Python:

.. code:: python
```

File: source/blueprints/overview.rst [hunk 8]
```diff
@@ -94,7 +106,31 @@ Build the Natural Language Processing models that power the app.
 3. Run
 ^^^^^^
 
-Interact with the app in the Python shell using the commands below. Try the queries shown in the examples, then try some queries of your own invention.
+Using command-line interface (CLI): you can use the command `python -m [blueprint's name] converse` to interact with the app.
```
Comment (ktick) [inline source/blueprints/overview.rst:109]:
```suggestion
Using CLI: use the command `python -m [bp_name] converse` to interact with the app.
```

File: source/blueprints/overview.rst [hunk 9]
```diff
@@ -94,7 +106,31 @@ Build the Natural Language Processing models that power the app.
 3. Run
 ^^^^^^
 
-Interact with the app in the Python shell using the commands below. Try the queries shown in the examples, then try some queries of your own invention.
```
Comment (ktick) [inline source/blueprints/overview.rst:97]: Interact with the app using the commands below. Try the queries shown in the examples, then try some queries of your own invention.

File: source/blueprints/overview.rst [hunk 10]
```diff
@@ -94,7 +106,31 @@ Build the Natural Language Processing models that power the app.
...
+   Retrieve object from KB: index= 'menu_items', id= 'B01N4VEUBF'.
+   Retrieve object from KB: index= 'menu_items', id= 'B01NCEYW8Z'.
+   Retrieve object from KB: index= 'menu_items', id= 'B01D8TCZUC'.
+   Retrieve object from KB: index= 'menu_items', id= 'B01MYFKNZW'.
+   Retrieve object from KB: index= 'menu_items', id= 'B01D8TCA48'.
+   App: Sure, I have 1 order of Pepperoni Pizza from Firetrail Pizza for a total price of $11.00. Would you like to place the order?
+   ...  Listening...
+
+
+Using Python: You can interact with the app with the following snippet. Try the queries shown in the examples, then try some queries of your own invention.
```
Comment (ktick) [inline source/blueprints/overview.rst:133]:
```suggestion
Using Python: use the following snippet to interact with the app.
```

**Comment by minhtuev** [comment]
Preview:

image

**Pull request merged**